.github/workflows/update_client.yaml

```diff
@@ -0,0 +1,95 @@
+name: Update SDK
+
+# Define the events that trigger the workflow
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "api/**"
+  pull_request:
+    types: [closed]
+    paths:
+      - "api/**"
+
+jobs:
+  run-commands:
+    # Define the OS and environment the job will run on
+    runs-on: ubuntu-latest
+
+    # Define the steps in the job
+    steps:
+      # Step 1: Checkout the first repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          repository: wordlift/docs
+          path: "./docs"
```

Comment:
I don't think you need this, being the current repo, e.g. https://github.com/wordlift/docs/blob/358834a2db5fd4aa7ff6b07f802b2b7903d75cff/.github/workflows/azure-static-web-apps-happy-pebble-0cf31410f.yml#L14